File: pm_core/src/core/domain/entity/priority_scheme_item_entity.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package entity

type PrioritySchemeItemEntity struct {
	BaseEntity
	TenantID   int64 `json:"tenantId"`
	SchemeID   int64 `json:"schemeId"`
	PriorityID int64 `json:"priorityId"`
	Sequence   int   `json:"sequence"`
}

File: pm_core/src/main/java/serp/project/pmcore/domain/priority/entity/PrioritySchemeItemEntity.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.pmcore.domain.priority.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;
import serp.project.pmcore.domain.shared.entity.BaseEntity;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class PrioritySchemeItemEntity extends BaseEntity {
    private Long tenantId;
    private Long schemeId;
    private Long priorityId;
    private Integer sequence;
}
